pom.xml

```diff
@@ -87,6 +87,19 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>cobertura-maven-plugin</artifactId>
```

Comment:
According to https://confluence.atlassian.com/display/CLOVER/Comparison+of+code+coverage+tools, the [JaCoCo plugin](http://www.eclemma.org/jacoco/trunk/doc/maven.html) is better then cobertura. Can you confirm? Can you switch to JaCoCo?